pom.xml

```diff
@@ -32,6 +32,7 @@
   </parent>
 
   <properties>
+    <project.version>1.2.0-SNAPSHOT</project.version>
```

Comment:
defining it as a property is interesting. not sure if there are disadvantages doing it this "variable" - looking fine i think